src/main.dev.js

```diff
@@ -20,6 +20,14 @@ import { autoUpdater } from 'electron-updater';
 import log from 'electron-log';
 import { configureStore } from '@reduxjs/toolkit';
 import { forwardToRenderer, triggerAlias, replayActionMain } from 'electron-redux';
+import {
+  MediaPlaybackStatus,
+  MediaPlaybackType,
+  SystemMediaTransportControlsButton,
+} from '@nodert-win10-au/windows.media';
+import { BackgroundMediaPlayer } from '@nodert-win10-au/windows.media.playback';
+import { RandomAccessStreamReference } from '@nodert-win10-au/windows.storage.streams';
+import { Uri } from '@nodert-win10-au/windows.foundation';
```

Comment:
Move top-level imports into the the `if (isWindows && isWindows10) block` L: 347
This is due to the build/runtime failing on Linux (and possibly macOS) due to not being able to install the native dependencies.

```javascript
if (isWindows) {
  const windowsMedia = require('@nodert-win10-au/windows.media');
  const windowsMediaPlayback = require('@nodert-win10-au/windows.media.playback');
  const windowsStorageStreams = require('@nodert-win10-au/windows.storage.streams');
  const windowsFoundation = require('@nodert-win10-au/windows.foundation');
 // ...rest of code
}
```